tests/suites/test_suite_mpi.function

```diff
@@ -1,5 +1,36 @@
 /* BEGIN_HEADER */
 #include "mbedtls/bignum.h"
+
+typedef struct mbedtls_test_mpi_random
+{
+    data_t *data;
+    size_t  pos;
+    size_t  chunk_len;
```

Comment:
You aren't using `pos` or `chunk_len`. Use them or lose them.

Reply:
I think it would be cleaner if we would use different structures for the two tests. (Unless of course if the random functions are identical too)

Reply:
ok, the random functions are actually not the same (mine is dumber, essentially memvopy with checks), so I will differentiate the structure as well.